src/backend/commands/analyze.c

```diff
@@ -1436,7 +1431,7 @@ static Oid buildSampleTable(Oid relationOid,
 /**
  * Drops the sample table created during ANALYZE.
  */
-static void dropSampleTable(Oid sampleTableOid)
+void dropSampleTable(Oid sampleTableOid, bool isExternal)
```

Comment:
We create several sample tables - one is a PXF table that is used to get the samples, and another is a AO table where the samples are stored.